src/test/rustdoc-gui/src/link_to_definition/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "link_to_definition"
+version = "0.1.0"
+edition = "2018"
+
+[lib]
+path = "lib.rs"
```

Comment:
There's no dependency on `implementors` in this `Cargo.toml`. I think you can just remove the `implementors` crate since it's not necessary for what we're testing?

Reply:
Oh damn, I added too much without paying attention, removed it!